### DISCUSSION TITLE
Move Ant tooling out of hibernate-core

### DISCUSSION THREAD

**Discussion by sebersole** [hibernate/hibernate-orm#4251] [Design Proposals]
I propose we move the Ant tasks out of `hibernate-core` and into `tooling/hibernate-ant` like the support for the other build tools.

Thoughts/objections?

**Comment by gavinking** [comment]
Definitely worth doing!

**Comment by sebersole** [comment]
https://hibernate.atlassian.net/browse/HHH-14848
